src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  if (days === 6) {
+    return 220;
```

Comment:
The return value for exactly 6 days is hard-coded to 220, which doesn't account for the `$20` discount for renting 3 or more days. The calculation should be dynamic based on the number of days.